cheatsheets/File_Upload_Cheat_Sheet.md

```diff
@@ -117,6 +117,12 @@ In order to avoid the above mentioned threat, creating a **random string** as a
 
 As mentioned in the [Public File Retrieval](#public-file-retrieval) section, file content can contain malicious, inappropriate, or illegal data.
 
+Based on the expected type, special file content validation can be applied:
+
+- For **images**, applying image rewriting techniques destroys any kind of malicious content injected in an image; this could be done through [randomization](https://security.stackexchange.com/a/8625/118367).
```

Comment:
Personally I do not like links to stackexchange/stackoverflow. I would prefer to describe `randomization` inside cheat sheet.
@rbsec @ThunderSon any thoughts about this?

Reply:
As far as I know, answers are never removed, they have a strong history of changes, and the answer provided good. If Robin agrees with you I'll definitely add them.

Reply:
I don't have  problem with links to stackexchange - and given this is a fairly minor point in the CS I'm not sure it's worth devoting a whole paragraph to trying to explain it.